packages/dso-toolkit/components/Componenten/definition-list/definition-list.config.yml

```diff
@@ -43,7 +43,7 @@ context:
       longread:
         - paragraph: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
         - paragraph: Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-    - term: "Pneumonoultramicroscopicsilicovolcanoconiosis:"
+    - term: "Pneumonoultramicroscopicsilicovolcanoconiosisfantasismus:"

```

Comment:
Stiekum ben ik ook nog wel benieuwd naar het resultaat:
- als er NA zo'n dt/dd combi met zo'n lange dt _nog_ een dt/dd komt;
- als er in een dt veel vulling staat, maar wel met normale vulling, dus woorden _gescheiden door spaties_

Reply:
2 dt-varianten toegevoegd